src/FsCheck/Arbitrary.fs

```diff
@@ -555,6 +563,32 @@ module Arb =
                             
             fromGenShrink (Gen.array2DOf generate,shrinkArray2D)
 
+        ///Generates arbitrary strings, which are lists of chars generated by Char.
+        static member String() = 
+            let generator = Gen.frequency [(9, Gen.map (fun (chars:char[]) -> new String(chars)) generate);(1, Gen.constant null)]
+            let shrinker (s:string) = 
+                    match s with
+                    | null -> Seq.empty
+                    | _ -> s.ToCharArray() |> shrink |> Seq.map (fun chars -> new String(chars))
+            fromGenShrink (generator,shrinker)
+
+        ///Generate an object - a boxed char, string or boolean value.
+        static member Object() =
```

Comment:
Yes. I'll revert that part of the change. I must've been on autopilot and thinking of reordering things as if they were let-bindings. They're all static members though so there's no need to reorder them.